examples/modules/core/plot_6_handle_times.py

```diff
@@ -0,0 +1,48 @@
+"""
+Handle time information
+=======================
+
+By default, :code:`spikeinterface` assumes that a recording is uniformly sampled and it starts at 0 seconds.
+However, in some cases there could be a different start time or even some missing frames in the recording.
+
+This notebook shows how to handle time informaiton in :code:`spikeinterface` recording and sorting objects.
+"""
+from spikeinterface.extractors import toy_example
+
+##############################################################################
+# First let's generate toy example with a single segment:
+
+rec, sort = toy_example(num_segments=1)
+
+
+##############################################################################
+# Now let's create a time vector by getting the default times and adding 5 s:
```

Comment:
Here I would add that noramly time vector is load by the extractor when the underlying format have a time vector but in our it is an external knowledge of time vector which is a special case.
No , What do yout think .?